src/test/java/com/ays/AbstractSystemTest.java

```diff
@@ -58,6 +61,14 @@ public void initializeAuth() {
                 .thenReturn(parameters);
         this.tokenConfiguration = new AysTokenConfigurationParameter(parameterService);
 
+        final Map<String, Object> claimsOfSuperAdminOne = new SuperAdminEntityBuilder()
+                .withId(AysTestData.SuperAdminUser.VALID_ID_ONE)
+                .withUsername(AysTestData.SuperAdminUser.VALID_USERNAME_ONE)
+                .withInstitutionId(AysTestData.SuperAdminUser.INSTITUTION_ID)
```

Comment:
Aşağıdaki şekilde olmasını önerebilirim;

```suggestion
                .withInstitutionId(AysTestData.Institution.VALID_ID_ONE)
```

Reply:
Super Admin'ın Institution ID null olması gerekir. AdminEntity'de o şekilde yapmıştık.

Reply:
Aslında setlemememiz null olduğunu gösterir, bu daha doğru olur

Reply:
Selamlar, düzelttim.